src/xrCore/Animation/Bone.hpp

```diff
@@ -303,9 +304,6 @@ class XRCORE_API CBone : public CBoneInstance, public IBoneData
     Fmatrix rest_transform;
     Fmatrix rest_i_transform;
 
-    // Fmatrix last_transform;
-
-    // Fmatrix render_transform;
```

Comment:
Return

Reply:
But you can remove additional blank line between them, like this:
```
    // Fmatrix last_transform;

    // Fmatrix render_transform;
```

Reply:
Restored.